### PR TITLE
Logger Updated

### PR DESCRIPTION
**Pull request by sadrasabouri**
### Changed
- `logger` function format for `erdos_reyni_gilbert` changed
- `log` function added

### CODE REVIEW

File: pyrgg/functions.py [hunk 1]
```diff
@@ -554,3 +555,25 @@ def check_for_config(input_func=input):
                 pyrgg.params.CONFIG_FILE_FORMAT.format("")):
             configs.append(file)
     return _print_select_config(configs, input_func)
+
+
+def log(file, file_name, elapsed_time, text):
```
Comment (sepandhaghighi) [inline pyrgg/functions.py:560]: I suggest `save_log` instead of `log`

File: pyrgg/engines/pyrgg.py [hunk 2]
```diff
@@ -265,23 +247,18 @@ def logger(file, file_name, elapsed_time, input_dict):
...
-                                          str(bool(input_dict["self_loop"])),
-                                          str(is_weighted(input_dict["max_weight"],
-                                                          input_dict["min_weight"],
-                                                          bool(input_dict["sign"]))),
-                                          str(input_dict["max_weight"]),
-                                          str(input_dict["min_weight"]),
-                                          input_dict["engine"],
-                                          ENGINE_MENU[input_dict["engine"]],
-                                          elapsed_time))
+        text = f"Vertices : {str(input_dict['vertices'])}\n"
```
Comment (sepandhaghighi) [inline pyrgg/engines/pyrgg.py:250]: I suggest using `.format()` instead of `f{}`

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

File: pyrgg/functions.py (hunk 1 above)
Comment (sadrasabouri) [reply pyrgg/functions.py:560]: ok, will apply

File: pyrgg/engines/pyrgg.py (hunk 2 above)
Comment (sadrasabouri) [reply pyrgg/engines/pyrgg.py:250]: The latest Python supporting version is `3.6` for pyrgg I guess we can use f-formats now.

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 - `pyrgg.engines.erdos_reyni` module
 ### Changed
+- `logger` function format for `erdos_reyni_gilbert` changed
+- `log` function added
```
Comment (sepandhaghighi) [inline CHANGELOG.md:12]: 1. `log` --> `save_log`
2. Move this line to `### Added` section

File: pyrgg/engines/pyrgg.py [hunk 4]
```diff
@@ -265,23 +247,18 @@ def logger(file, file_name, elapsed_time, input_dict):
...
+        text += f"Total Edges : {str(input_dict['edge_number'])}\n"
```
Comment (sepandhaghighi) [inline pyrgg/engines/pyrgg.py:251]: `.format()` instead of `f{}`

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@sadrasabouri 
Take a look at my comments 💯

File: pyrgg/engines/pyrgg.py (hunk 4 above)
Comment (sadrasabouri) [reply pyrgg/engines/pyrgg.py]: fixed in [8a01d50](https://github.com/sepandhaghighi/pyrgg/pull/182/commits/8a01d50e5dbe80f3414060a9ef679186263768eb)

File: CHANGELOG.md (hunk 3 above)
Comment (sadrasabouri) [reply CHANGELOG.md]: fixed in [631408e](https://github.com/sepandhaghighi/pyrgg/pull/182/commits/631408ee88efb334f5c86e64e5262b13ccdc7549)

**Comment by sadrasabouri** [comment]
Comments are fixed, sorry for missing the `pyrgg` engine's f-format. Please re-review again @sepandhaghighi

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 ### Added
 - `pyrgg.engines.erdos_reyni` module
+- `logger` function format for `erdos_reyni_gilbert` changed
+- `save_log` function added
```
Comment (sepandhaghighi) [inline CHANGELOG.md:11]: - `save_log` function

File: CHANGELOG.md [hunk 6]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 ### Added
 - `pyrgg.engines.erdos_reyni` module
+- `logger` function format for `erdos_reyni_gilbert` changed
```
Comment (sepandhaghighi) [inline CHANGELOG.md:10]: Please move this to the `Changed` section

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

**Pull request merged**